README.md

```diff
@@ -1,12 +1,25 @@
-HypothesisTests.jl
-===========
+## HypothesisTests.jl
 
-[![Build Status](https://travis-ci.org/JuliaStats/HypothesisTests.jl.svg?branch=master)](https://travis-ci.org/JuliaStats/HypothesisTests.jl)
-[![Coverage Status](https://coveralls.io/repos/JuliaStats/HypothesisTests.jl/badge.svg?branch=master)](https://coveralls.io/r/JuliaStats/HypothesisTests.jl?branch=master)
-[![HypothesisTests](http://pkg.julialang.org/badges/HypothesisTests_0.5.svg)](http://pkg.julialang.org/?pkg=HypothesisTests)
-[![HypothesisTests](http://pkg.julialang.org/badges/HypothesisTests_0.6.svg)](http://pkg.julialang.org/?pkg=HypothesisTests)
+*HypothesisTests.jl* is a Julia package that implements a wide range of hypothesis tests.
 
-This package implements several hypothesis tests in Julia.
+- **Current Release**:
+  [![HypothesisTests](http://pkg.julialang.org/badges/HypothesisTests.5.svg)
```

Comment:
Since we no longer support 0.5 as of #108, this badge could be dropped. This is more of a note-to-self; you don't have to do that here.

Reply:
Well, that should probably wait anyway until we have a 0.6-only release.